docs/modules/ROOT/pages/client.adoc

```diff
@@ -163,6 +163,10 @@ include::./includes/additional-properties-as-attribute.adoc[leveloffset=+1, opts
 
 include::./includes/additional-request-args.adoc[leveloffset=+1, opts=optional]
 
+[[use-bean-validation]]
+== Add Bean Validation support to every API method and validation annotations to all models
```

Comment:
It seems like you forgot to add a title here. Should `Add Bean Validation support to every API method and validation annotations to all models` be the paragraph?

Reply:
I tried to add the description of bean validation like the above additional-request-args. So this should be the paragraph.